test/parallel/test-process-warnings.mjs

```diff
@@ -0,0 +1,116 @@
+import { spawnPromisified } from '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import { describe, it } from 'node:test';
+import { strictEqual, match, doesNotMatch } from 'node:assert';
+
+const fixturePath = fixtures.path('disable-warnings.js');
+const dep0025Message = /\(node:\d+\) \[DEP0025\] DeprecationWarning: sys is deprecated\. Use util instead\./;
+const dep0094Message = /\(node:\d+\) \[DEP0094\] DeprecationWarning: assert\.fail\(\) with more than one argument is deprecated\. Please use assert\.strictEqual\(\) instead or only pass a message\./;
+const experimentalWarningMessage = /\(node:\d+\) ExperimentalWarning: vm\.measureMemory is an experimental feature and might change at any time/;
+
+describe('process warnings', { concurrency: true }, () => {
+
+  it('should emit all warnings by default', async () => {
+    const { stdout, stderr, code, signal } = await spawnPromisified(process.execPath, [
+      fixturePath,
+    ]);
+
+    strictEqual(stdout, '');
+    match(stderr, dep0025Message);
```

Comment:
Am I the only one who thinks `assert.strictEqual` and `assert.match` are much more readable than lone `strictEqual` and `match`?

Reply:
I agree with @aduh95. It's really easy to loose the context when reading this. For example `match` is too generic.

Reply:
done 👍